packages/sku/src/types/types.ts

```diff
@@ -128,6 +128,32 @@ export interface SkuConfig {
    * @default: "jest"
    */
   __UNSAFE_EXPERIMENTAL__testRunner?: 'vitest' | 'jest';
+  /**
+   * An array of cjs dependencies that vite should treat as ESM.
+   * This is an experimental option that may change or be removed without notice.
```

Comment:
This message and the one in the changeset could be more accurate. See [the docs](https://github.com/cyco130/vite-plugin-cjs-interop#:~:text=automatically%20unwrap%20the%20default%20export%20from%20cjs) for the plugin.